<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.3">
    <title>Links for face-recognition-atd</title>
  </head>
  <body>
    <h1>Links for face-recognition-atd</h1>
<a href="https://files.pythonhosted.org/packages/ee/cd/8675cecc55375fe4e701762f5674ea824f091c99c3f97b69eb82acfa5ab9/face_recognition_atd-2.0.0-py3-none-any.whl#sha256=2e0f86c675122c09472a5f4d25a2877868c93f1889e26887bb22814cd3796e8b" data-dist-info-metadata="sha256=dc4604737fddfb73713ac47c07386f0567b79a855f789b5528733ab51d40190c" data-core-metadata="sha256=dc4604737fddfb73713ac47c07386f0567b79a855f789b5528733ab51d40190c">face_recognition_atd-2.0.0-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/99/1c/39114be51bf755c142a1cf31bc8a264c58501a4642660a008692f616ce5e/face_recognition_atd-2.0.0.tar.gz#sha256=2fc52c41eae99ffc736ee555913676b068f35cf37045c83e4fec3e51faafdd0a" >face_recognition_atd-2.0.0.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/f4/b3/e7a90ce9fdb7c57b845f9366588a92cb4d15f9d8902c5c4ecab28516fc17/face_recognition_atd-2.0.1-py3-none-any.whl#sha256=6ec7de7b6f235c1261cdd3ff743a8ec963bbfbb539531d572beb851ddcb2c1fa" data-dist-info-metadata="sha256=599f5469e4e57ec206e7b2506088304b9931b176dcf0b739ae7a52af13a0970d" data-core-metadata="sha256=599f5469e4e57ec206e7b2506088304b9931b176dcf0b739ae7a52af13a0970d">face_recognition_atd-2.0.1-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/d9/d4/ec84a681251cf7723dbd53ff5278fbcbe91498a41b31c44987953d982834/face_recognition_atd-2.0.1.tar.gz#sha256=7e0a721085849fb1ad2eb5de58f23b14bb341b04f71bf725052c3970e8612936" >face_recognition_atd-2.0.1.tar.gz</a><br />
</body>
</html>
<!--SERIAL 27171298-->